ROADMAP.md

```diff
@@ -0,0 +1,171 @@
+  - Guide on compiling/publishing ES2015+, .mjs, etc: https://twitter.com/philwalton/status/908082461799616512
+  - Support multi-build/folder outputs?
+
+### Codemods for TC39 Proposals
+
+> Lebab/others are already used to convert from ES5 -> ESNext, so incorporate it into Babel itself.
+
+- Refactor [Lebab](https://github.com/lebab/lebab/issues/138) as Babel transforms (can keep the cli since it's a separate tool)
+  - Usecase: ES3 -> ES6+ (on source code)
+  - Usecase: Remove usage of dropped proposals
+  - Usecase: Auto upgrade to the latest version of a proposal spec (if possible)
+  - Can we somehow combine forces in: babel-codemod/jscodeshift/lebab, prettier/recast/babel-generator? I really don't want to update all of these: new syntax equals re-writing the printer in all of these places separately/out of sync.
+
+## Increasing the quality of community plugins
+- Work with the community to create guides on how to write plugins or understand ASTs, etc
+- Analysis of API's/syntax used (Google BigQuery)
+- Have #blessed/sanctioned/curated packages according to some standard
+  - Can use for smoke tests
+  - Official testing package
+  - Certain level of coverage, downloads, etc
+  - Create a scoped namespace on github/npm for these? like webpack-contrib
+  - Can enforce linting rules on apis?
+  - Makes ecosystem changes easier if can notify and upgrade these plugins beforehand
+  - Create a set of standard tests to verify against (handles all syntax)
+  - Documented/tested set of options
+
+### ASTExplorer
+- allow custom version of babel-standalone (same as REPL to allow per PR tests)
+- integrate with repl? (both are in react)
+- auto publish a plugin to npm?
+- create tests?
+- hook into codesandbox?
+
+---
+
+## Feature
+
+### Performance table for ES6+ (used in babel-preset-env under new option e.g `perf`)
+
+> Add a new option in `preset-env`: will continue to compile if the native support is *slower*. Can setup a threshold, may need to compare against the size difference.
+
+- Use [six-speed](https://github.com/kpdecker/six-speed) repo as a base, needs to apply for ES6 and proposals
+- Need continued maintainence
+
+### Compiled Output Stats 
+
+> [#5340](https://github.com/babel/babel/issues/5340) Can show the code size before and after compiling to give a sense of compiled output. Could create suggestions like using "loose" mode or not compiling, etc.
+
+- The [REPL](https://twitter.com/existentialism/status/948940160653291520) just added a before/after code-size
+- Maybe difficult to do per transform
+
+### Async Transforms
+
+Support having async plugins. Will require it to be opt-in and for all other plugins to be async.
+
+### Plugin Ordering
+
+- Add `before`/`after` keys for plugins so they can specify order.
+- Possibily implement related plugins in the same "plugin" but just expose a flag out to the end-user.
+
+### Babel Helpers (shared code)
+- loader should handle these automatically like rollup
```

Comment:
maybe actually the easiest way for this is just to publish helpers as a package (they are somewhat already published within `@babel/runtime`) and encourage transpiling libraries (making it default maybe?) with imports being added to that package instead of inlining them into the source code like now